openassessment/xblock/static/sass/oa/elements/_staff.scss

```diff
@@ -44,6 +61,13 @@
     color: $copy-staff-color;
   }
 
+  .ui-staff__subtitle {
+    @extend %t-subheading;
+    @include fontSize($f-size-medium);
+    font-weight: 600;
```

Comment:
Found the variable for this: 
```
font-weight: %t-strong;
```
also add this inside the rule: 
```
span {
font-weight: inherit;
}
```
since you need to override the normal font-weight set on all spans.